test/fixtures/full_config.yml

```diff
@@ -4379,7 +4379,7 @@ Style/StringHashKeys:
 Style/StringLiterals:
   Description: Checks if uses of quotes match the configured preference.
   StyleGuide: "#consistent-string-literals"
-  Enabled: false
+  Enabled: true
```

Comment:
I have always despised this rule but I'm not gonna spend any more energy fighting it

Reply:
I'll keep this out